README.md

```diff
@@ -8,6 +8,14 @@
 - SQL
 
```

Comment:
- [ ] Your frontend app has no tests.  Kindly add at least some tests to ensure your app's integrity.

As mentioned in the backend PR, it is very important for you to become familiar with RSPEC and JEST here at Microverse, and not on your new job.  Also, in technical interviews, you'll be required to write tests (I speak from experience).

That way, your project will be ready for presentation, and you will be more than ready for the software industry 💪

Reply:
Done ✔️